pipeline/scripts/remove-pkg.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+const child_process = require('child_process');
```

Comment:
It would maybe be nice to leave a comment in this file describing what its purpose is and why it's necessary